config/locales/en.yml

```diff
@@ -645,6 +644,10 @@ en:
           attributes:
             employing_school_id:
               blank: You must choose a school
+            results_search_again_query:
+              blank: You much enter a search term
+            no_results_search_again_query:
+              blank: You much enter a search term
```

Comment:
![image](https://user-images.githubusercontent.com/470137/116687977-ec127100-a9ad-11eb-9494-abb91153ef52.png)

Reply:
Can't believe I missed that - good spot! 😄